### DISCUSSION TITLE
Question: is UAC supposed to break clipboard and drag&drop?

### DISCUSSION THREAD

**Discussion by FrayxRulez** [microsoft/WindowsAppSDK#381] [Q&A]
Hello,
In the previous months I received few reports from users complaining about my app not being able to handle files pasted and dropped from Explorer and others.
Now, after a quite long research I figured out that this happens when UAC is disabled for logged in user.
Some of the reporters sustain that they always had UAC turned off, but that this problem started to happen only at some point, but of course I can't confirm this.

Now my question is: is this the intended behavior?
From my point of view this just creates a lot of confusion users have absolutely no hint about this, and the app is just broken for them.

**Comment by stevewri** [comment]
Clarifying: "UAC disabled" means changing the UAC slider to "never notify"?

**Comment by stevewri** [comment]
@ieburk: are you aware of any issues with file paste or drop with UAC prompts disabled?

**Comment by FrayxRulez** [comment]
Hi, yes, I meant "never notify"!

**Comment by FrayxRulez** [comment]
I forgot to mention that this is about UWP, as I'm used to post in microsoft-ui-xaml repository...

**Comment by prayaas-a** [comment]
Not sure if this is exactly helpful as I don't have repro steps at the moment (but can try to come up with them) but I have experienced issues with Clipboard working across UWP/Win32 in general a lot where things copied from UWP won't paste to the non-UWP world or vice versa. It's as if there's multiple clipboards.

**Comment by FrayxRulez** [comment]
Actually I'm not even sure that UAC is the root cause, but it seems to be at least one of them.
The problem is that dealing with final users it is super hard for me to get more details and being more specific/able to offer repro steps.

This is what I usually receive as report: [video](https://media.giphy.com/media/FtVSdaewiKNdFNKLyz/source.mp4).
The interesting part is that the 🚫 doesn't look generated by the UWP platform (as it would be a white tooltip with red crossed circle).

Usually users have UAC disabled, and often (not always) enabling it allows them to drop and paste files after.
